podman.hcl

```diff
@@ -2,6 +2,7 @@ description = "A tool for managing OCI containers and pods."
 sha256-source = "https://github.com/containers/podman/releases/download/v${version}/shasums"
 binaries = ["podman"]
 source = "https://github.com/containers/podman/releases/download/v${version}/podman-remote-static-linux_${arch}.tar.gz"
+requires = [ "gvproxy" ]
```

Comment:
Ignoring the other issues, I won't merge this change. We don't want to set opinionated defaults in packages.